Gemfile

```diff
@@ -17,6 +17,8 @@ else
   gem 'journey'
 end
 
+gem 'i18n', '~> 0.6.11'
```

Comment:
They will, because the interpolation backport has been removed. We may need to bring that backport back entirely if we want that to happen (even though I18n 0.7 is officially not supporting such rubies anymore).